index.html

```diff
@@ -0,0 +1,172 @@
+          </section>
+
+          <!-- Feature section -->
+  <div class="container">
+    <!-- <ul>
+      <li>Feature</li>
+      <li>home page</li>
+    </ul> -->
+    <section id="features">
+      <div class="grid">
+        <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+        <div class="desc">
+          <h2>Premium Materials</h2>
+          <p>
+            Our trombones use the shiniest brass which is sourced locally. This
+            will increase the longevity of your purchase.
+          </p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+        <div class="desc">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished
+            making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+        </div>
+        <div class="desc">
+          <h2>Quality Assurance</h2>
+          <p>
+            For every purchase you make, we will ensure there are no damages or
+            faults and we will check and test the pitch of your instrument.
+          </p>
+        </div>
+      </div>
+        </section>
+            <section id="how-it-works">
+            <iframe
+                id="video"
+                height="315"
+                src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+                allowfullscreen
+            ></iframe>
+            <!-- <p>images</p> -->
+            </section>
+            <section id="pricing">
+            <div class="product" id="tenor">
+                <div class="level">Tenor Trombone</div>
+                <h2>$600</h2>
+                <ol>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+                </ol>
+                <button class="btn">Select</button>
+            </div>
+            <div class="product" id="bass">
+                <div class="level">Bass Trombone</div>
+                <h2>$900</h2>
+                <ol>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+                </ol>
+                <button class="btn">Select</button>
+            </div>
+            <div class="product" id="valve">
+                <div class="level">Valve Trombone</div>
+                <h2>$1200</h2>
+                <ol>
+                <li>Plays similar to a Trumpet</li>
+                <li>Great for Jazz Bands</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+                <!-- <li>Lorem ipsum. dolor.</li> -->
```

Comment:
Do not use unused code as a comment in your codebase. Unused code as comments makes your code not clean

Reply:
- [ ] OPTIONAL: As this is a review practice project, I was not expecting two same inline comments. But I think you have got the whole idea and I will leave this comment as optional. :pray: